tests/test_middleware.py

```diff
@@ -108,6 +108,19 @@ def test_fallback_with_evil_host(self):
         response = self.client.get('/', HTTP_HOST='evil.com')
         self.assertEqual(response.status_code, 400)
 
+    @override_settings(
+        ROOT_HOSTCONF='tests.hosts.simple',
+        DEFAULT_HOST='www',
+        ALLOWED_HOSTS=['somehost.com'],
+        DEBUG=False)
+    def test_fallback_with_evil_host_response(self):
```

Comment:
This test doesn't make much sense to me, but it's the only way I found to exercise the `except DisallowedHost` in `HostsResponseMiddleware`.  This test, instead of running a disallowed host request through both the request and response middleware (as would happen in a real project), only runs the response middleware.

If I include the request middleware as well, the `DisallowedHost` is raised in the request middleware, the exception is caught, and the fallback happens without the response middleware `try` block ever running.  This is why that `try` block is not tested in the existing tests.

Because the response middleware just returns the response after catching `DisallowedHost`, I don't see much else we can do in the test other than asserting we get back what we put in the argument.

Reply:
How about removing the seemingly obsolete exception catching as in https://github.com/jazzband/django-hosts/pull/114?

Reply:
The `except` block was still needed for Django 1.11, which we hadn't dropped support for until 26 minutes ago.  Now that we have https://github.com/jazzband/django-hosts/pull/114, I agree.